sdk/core/core-tracing/lib/implementations/noop/tracerNoOpImpl.ts

```diff
@@ -2,8 +2,12 @@ import { Tracer } from "../../interfaces/tracer";
 import { SpanOptions } from "../../interfaces/SpanOptions";
 import { Span } from "../../interfaces/span";
 import { SpanNoOpImpl } from "./spanNoOpImpl";
+import { SupportedPlugins } from '../../utils/supportedPlugins';
 
 export class TracerNoOpImpl implements Tracer {
+  public get pluginType() : SupportedPlugins {
```

Comment:
Why is this a getter?

Reply:
readonly field would probably be more appropriate?

Reply:
fixed with https://github.com/Azure/azure-sdk-for-js/pull/4668/commits/66e6792f51af329205f8b1edefa34bc3490c308d